src/core/farcasterTypes.ts

```diff
@@ -61,9 +61,14 @@ export type FrameButtonMetadata = {
   action?: 'post' | 'post_redirect';
 };
 
+export type FrameInputMetadata = {
+  prompt_text: string;
```

Comment:
I think is best to keep this as close to the original API, so have `prompt_text` as `text`.